lib/taiko.js

```diff
@@ -620,6 +678,32 @@ module.exports.focus = async (selector, options = {}) => {
     return { description: 'Focussed on the ' + description(selector, true) };
 };
 
+/**
+ * Override browser permissions
+ * 
+ * @example
+ * overridePermissions('http://maps.google.com',['geolocation']); 
+ * 
```

Comment:
Adding reference for all available permissions that can be overridden will be helpful